src/modules/dashboard/components/ColonyMembers/ColonyMembers.tsx

```diff
@@ -108,6 +113,25 @@ const ColonyMembers = () => {
     });
   }, [openPermissionManagementDialog, colonyData, isVotingExtensionEnabled]);
 
+  const openToggleManageWhitelistDialog = useDialog(ManageWhitelistDialog);
+
+  const handleToggleWhitelistDialog = useCallback<
+    MouseEventHandler<HTMLButtonElement>
+  >(
+    (evt) => {
+      evt.stopPropagation();
+      /*
+       * We don't have, and can't inject all the required props that the component
+       * is expecting when using it in a wizard
```

Comment:
So maybe they are not required actually? if we can have both cases

Reply:
Good point, I will give it another try.